# perception list, defined by ros message type
Header header
Localization pose # localization info
PerceptionMsg[] perceptions #percetion Array

